### ISSUE TITLE
In 'The Fast and the Furious', the screen is controlled, not the car.

### ISSUE THREAD

**Issue by Jmousy** [ruffle-rs/ruffle#4117]
**Describe the bug**
[The Fast and the Furious.zip](https://github.com/ruffle-rs/ruffle/files/6371788/The.Fast.and.the.Furious.zip)
![fatf](https://user-images.githubusercontent.com/75052171/115991380-055c9b80-a603-11eb-9dbb-9803bc8d4b37.gif)

(Refer to the attached gif at the top)

After starting the game in 'The Fast and the Furious', if you hold down the arrow keys up or down, the entire screen will move up or down, not the car. Soon, the entire screen disappears and the game cannot be played.

**Expected behavior**
The car should move, not the screen.

**Is the problem with the Ruffle desktop app, extension, or self-hosted version?**
Checked in self-hosted version.

**What platform are you using?**
 - Windows 10 20H1 x64
 - Google Chrome (latest)
 - Ruffle Nightly 2021-04-25

**Comment by Toad06** [comment]
Thanks for the report.

The code related to keyboard controls is located in the inline function `movieclip.prototype.bgcontrol`. I've added `trace(this);` in this function and it returns "_level0.road" in Flash and Ruffle.

Going a bit further in the code, we can see different calls to the function `rotate`. Actually this is a method:
```
movieclip.prototype.rotate = function()
{
   _Y = _Y + Math.cos(0.017453292519943295 * _rotation) * _root.w;
   _X = _X - Math.sin(0.017453292519943295 * _rotation) * _root.w;
};
```
I've added `trace(this);` in this inline function and it returns "_level0.road" in Flash, "_level0" in Ruffle.
So the _x and _y being changed in Ruffle are the ones of the root movie, causing the issue.

Worth mentioning that changing `rotate();` to `this.rotate();` makes the game work in both Flash and Ruffle.
